### PR TITLE
Stepwise Regression Example Added

### PR DESCRIPTION
**Pull request by falconcode16**
### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by mrmaxguns** [comment]
Python filenames should end with the extension `.py`. The filename should be in snake_case, i.e. `stepwise_regression.py`. The file bypassed all checks since it wasn't recognized as a Python file.

File: machine_learning/Stepwise Regression [hunk 1]
```diff
@@ -0,0 +1,64 @@
+from sklearn.datasets import load_boston
+import pandas as pd
+import numpy as np
+import statsmodels.api as sm
+
+data = load_boston()
```
Comment (mrmaxguns) [inline machine_learning/Stepwise Regression:6]: You should put the test dataset and `X` and `y` variables in an `if __name__ == "__main__"` block at the end of the file:

```python
# End of file
if __name__ == "__main__":
    data = load_boston()
    X = pd.DataFrame(data.data, columns=data.feature_names)
    y = data.target

    result = stepwise_selection(X, y)

    print('resulting features:')
    print(result)

File: machine_learning/Stepwise Regression [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+X = pd.DataFrame(data.data, columns=data.feature_names)
+y = data.target
+
+
+def stepwise_selection(X, y,
```
Comment (mrmaxguns) [inline machine_learning/Stepwise Regression:11]: Add type hints to the function https://docs.python.org/3/library/typing.html

File: machine_learning/Stepwise Regression [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            if verbose:
+                print('Drop {:30} with p-value {:.6}'.format(worst_feature, worst_pval))
+        if not changed:
+            break
+    return included
+
+result = stepwise_selection(X, y)
+
+print('resulting features:')
+print(result)
```
Comment (mrmaxguns) [inline machine_learning/Stepwise Regression:64]: Add a newline at the end of the file. Black should do this for you.

**Review by mrmaxguns — CHANGES_REQUESTED** [review]

**Comment by dhruvmanila** [comment]
Please read the [Contributing guidelines](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md) before opening a pull request.

Closing this PR as it contains an extensionless file.

**Pull request closed without merging**